packages/gestalt/src/ModuleExpandable.js

```diff
@@ -23,9 +25,13 @@ export default function ModuleExpandable({
   id,
   accessibilityExpandLabel,
   accessibilityCollapseLabel,
+  expandedId,
+  setExpandedId,
   items,
 }: Props): Node {
-  const [expandedId, setExpandedId] = useState(-1);
+  const [localExpandedId, setLocalExpandedId] = useState(null);
+  const expId = expandedId || localExpandedId;
```

Comment:
Since we're using a number, we'll want to change this to use nullish coalescing so we don't ignore 0:
```js
const effectiveExpandedIndex = externalExpandedIndex ?? expandedIndex;
```